### PR TITLE
Update be.m3u

### PR DESCRIPTION
**Pull request by bhedouin**
- update Canal Zoom and Antenne Centre

### CODE REVIEW

File: streams/be.m3u [hunk 1]
```diff
@@ -3,8 +3,8 @@
 https://bel-lh.akamaihd.net/i/BEL_1@321282/master.m3u8
 #EXTINF:-1 tvg-id="BX1.be" status="online",BX1 (720p) [Not 24/7]
 https://59959724487e3.streamlock.net/stream/live/playlist.m3u8
-#EXTINF:-1 tvg-id="CanalZoom.be" status="error",Canal Zoom (720p) [Not 24/7]
-http://streamer.canalc.be:1935/canalzoom/smil:SMIL-canalzoom-multi/playlist.m3u8
+#EXTINF:-1 tvg-id="CanalZoom.be" status="online",Canal Zoom (720p) [Not 24/7]
+https://tvlocales-live.freecaster.com/live/95d2e3af-5ab8-45a9-9dc9-f544d006b5d5/95d2e3af-5ab8-45a9-9dc9-f544d006b5d5.isml/95d2e3af-5ab8-45a9-9dc9-f544d006b5d5-audio_und=128000-video=2600000.m3u8
```
Comment (LaneSh4d0w) [inline streams/be.m3u:7]:
```suggestion
https://tvlocales-live.freecaster.com/live/95d2e3af-5ab8-45a9-9dc9-f544d006b5d5/95d2e3af-5ab8-45a9-9dc9-f544d006b5d5.isml/master.m3u8
```

File: streams/be.m3u [hunk 2]
```diff
@@ -63,5 +63,5 @@ https://live-radio-cf-vrt.akamaized.net/groupb/live/eee20dc8-158a-4194-8d92-0c5a
 https://rtltvi-lh.akamaihd.net/i/TVI_1@319659/master.m3u8
 #EXTINF:-1 tvg-id="" status="online",TVO (576p)
 https://media.mediahuisvideo.be/hls/account=tqIhfOEY3U0A/item=oIFtpshSvwcy/oIFtpshSvwcy.m3u8
-#EXTINF:-1 tvg-id="" status="error",Antenne Centre (720p) [Geo-blocked]
-https://actv-live.freecaster.com/live/actv/actv.m3u8
+#EXTINF:-1 tvg-id="" status="online",Antenne Centre (720p) [Geo-blocked]
+https://tvlocales-live.freecaster.com/live/95d2f733-1bc2-4bd5-9b96-c53ed3fc450f/95d2f733-1bc2-4bd5-9b96-c53ed3fc450f.isml/95d2f733-1bc2-4bd5-9b96-c53ed3fc450f-audio_und=128000-video=5200000.m3u8
```
Comment (LaneSh4d0w) [inline streams/be.m3u:67]:
```suggestion
https://tvlocales-live.freecaster.com/live/95d2f733-1bc2-4bd5-9b96-c53ed3fc450f/95d2f733-1bc2-4bd5-9b96-c53ed3fc450f.isml/master.m3u8
```

File: streams/be.m3u (hunk 1 above)
Comment (baptiste313) [reply streams/be.m3u:7]: It works in France

**Review by its0x08 — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
i think a mistake was made.

**Comment by freearhey** [comment]
Closed due to lack of activity.

**Pull request closed without merging**